_posts/2020-09-15-open-sourcing-a-private-project.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "How to Open Source a Closed Source Project"
```

Comment:
"Private Project" sounds more common to me than "Closed Source Project". But either one looks good 👍